bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggHCPjets_v3/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
do you mind removing these default settings?

Reply:
Done for all *proc_card.dat files in four processes. Pushed to git.